src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logoImgLink" href="#home">
+        <img 
+          class="logoImg" 
+          src="./images/logo.png" 
+          alt="logo"
+        >
+      </a>
+
+      <nav class="navBlock">
+        <ul class="navBlock__items">
+          <li class="navBlock__item">
+            <a class="navBlock__link is-active" href="#">Apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
```
Такі назви більш звичні